spec/features/hub/edit_organization_spec.rb

```diff
@@ -22,13 +22,15 @@
 
         select "Central Time (US & Canada)", from: "Timezone"
         fill_in "Capacity limit", with: "200"
+        check "Allows Greeters"
```

Comment:
I think this will have no effect (it'll already be checked) since the default is true.

Reply:
Oh good catch! Yeah my bad I made these changes before the default was set to true